vendor/engines/bulk_email/app/controllers/bulk_email/bulk_email_controller.rb

```diff
@@ -21,15 +21,12 @@ class BulkEmailController < ApplicationController
     def search
       @searcher = BulkEmailSearcher.new(@search_fields)
       @users = @searcher.do_search
+    end
 
-      respond_to do |format|
-        format.html
-
-        format.csv do
-          filename = "bulk_email_#{params[:bulk_email][:user_types].join('-')}.csv"
-          set_csv_headers(filename)
-        end
-      end
+    def download
+      @users = User.where_ids_in(params[:recipient_ids])
```

Comment:
I don't think it's anything newer than that, so I think we're probably ok. I wouldn't worry about it. I just wanted to raise the idea.